src/api/app/controllers/webui/cloud/upload_jobs_controller.rb

```diff
@@ -53,7 +60,7 @@ def validate_uploadable
         redirect_to cloud_upload_index_path
       end
 
-      def validate_configuration_presence
+      def validate_ec2_configuration_presence
```

Comment:
The question is: How are you planning to implement the new and create actions for Azure? If they will get implemented in the same actions as EC2, drop this function and always redirect to the static configuration page. If you plan to split these actions, then leave it.

Reply:
I didn't decide this, because this was not part of the card I'm working on and I don't see the need to decide that now as it's not relevant for this UI refactoring.

Reply:
Well, then we can drop the whole PR more or less because the idea is to not change the workflow afterwards. However, if you decide to do upload in the same actions (I think I would prefer this) you would need to adapt this which implies again a workflow change. So I would prefer to decide now.